lineage/data/Lineage_collections.py

```diff
@@ -106,8 +94,7 @@ def popout_single_lineages(lineages):
 gem06 = [LineageTree(list_of_cells, E) for list_of_cells in import_Heiser(path=r"lineage/data/heiser_data/new_version/AU00801_A3_1_V4.xlsx")]
 gem062 = [LineageTree(list_of_cells, E) for list_of_cells in import_Heiser(path=r"lineage/data/heiser_data/new_version/AU00801_A3_2_V4.xlsx")]
 
-gemControl = gem04 + gem05 + gem06 + gem042 + gem052 + gem062 + gem043
-Gemcitabine_Control = popout_single_lineages(gemControl)
```

Comment:
Why did you remove `popout_single_lineages`? I'd guess that's why this is failing.

Reply:
I tried bringing it back, still got the error.
I thought removing single lineages means removing those that have phase extension. I can remove them if it doesn't create some bias.

Reply:
When I was debugging the test I realized that we have some cases like:
[nan, nan, 62.5, nan, 0, nan]
which means we don't know about G1 and G2 fates,
the G1 is censored and the length of it is 62.5
we have not observed G2.

Is this something feasible? I mean the experiment time is 96 hours, and the cell is censored in G1, and we don't know about its fate. I mean in this case, shouldn't we have 96 instead of 62.5?

Reply:
And this is a root cell? If so, yes, something is off about this. If you can trace it back to a specific file it'd be helpful to see what's there. What does the 0 represent?